src/clients/api/mutations/supplyBnb.ts

```diff
@@ -0,0 +1,32 @@
+import Web3 from 'web3';
+import { TransactionReceipt } from 'web3-core';
+import { getVBepToken } from 'utilities';
+
+export interface ISupplyBnbInput {
+  tokenContract: $TSFixMe; // @TODO: use contract type once defined (Typechain?)
+  web3: Web3;
+  account: string | undefined;
+  amount: string;
+}
+
+const vBnbAddress = getVBepToken('bnb').address;
+
+export type SupplyBnbOutput = TransactionReceipt;
+
+const supplyBnb = async ({
+  web3,
+  tokenContract,
+  account,
+  amount,
+}: ISupplyBnbInput): Promise<SupplyBnbOutput> => {
+  const contractData = tokenContract.methods.mint().encodeABI();
+  const tx = {
+    account,
+    to: vBnbAddress,
+    value: amount.toString(),
```

Comment:
```suggestion
    value: amount.toFixed(),
```

So I found out yesterday that the `toString` method will write large numbers just like they're written in JavaScript: 1e20 for example. I've had a few issues with it on the front end already, so it might be safer to use `toFixed()` instead.